status.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/hansmi/paperhooks/pkg/client"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type statusClient interface {
+	GetStatus(ctx context.Context) (*client.SystemStatus, *client.Response, error)
+}
+
+type statusCollector struct {
+	cl statusClient
+
+	storageTotalDesc                *prometheus.Desc
+	storageAvailableDesc            *prometheus.Desc
+	databaseStatusDesc              *prometheus.Desc
+	databaseUnappliedMigrationsDesc *prometheus.Desc
+	redisStatusDesc                 *prometheus.Desc
+	celeryStatusDesc                *prometheus.Desc
+	indexStatusDesc                 *prometheus.Desc
+	indexLastModifiedDesc           *prometheus.Desc
+	classifierStatusDesc            *prometheus.Desc
+	classifierLastTrainedDesc       *prometheus.Desc
+}
+
+// Only doing this to be able to unit test the collector
+var timeSince = time.Since
+
+func newStatusCollector(cl statusClient) *statusCollector {
+	return &statusCollector{
+		cl: cl,
+
+		storageTotalDesc:                prometheus.NewDesc("paperless_status_storage_total", "Total storage of Paperless in bytes.", nil, nil),
```

Comment:
Please add the unit as a suffix, i.e. `_bytes`.

https://prometheus.io/docs/practices/naming/

Reply:
Fixed in https://github.com/hansmi/prometheus-paperless-exporter/pull/58/commits/9d3636717206fdf9cd5fb8f543b40281bdaee0af